sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
+    
+    function __construct() {
+    }
+
+    private function get_settings($var_name, $default_value) {
+        if (isset($this->$var_name)) {
+            return $this->$var_name;
+        } else {
+            return $default_value;
+        }
+    }
+
+    function missing_person() {
```

Comment:
But if we have a hundred config params, will we have hundred function here!?

Yeah, there is no problem in default being of any datatype. What I was proposing was, set everything which is not manually overridden in `config.php` to its default value in the constructor of this lib.

Anyway, if you still are in favor of this, I am fine with this.